core/src/main/scala/io/odin/formatter/Formatter.scala

```diff
@@ -16,31 +17,28 @@ object Formatter {
 
   val BRIGHT_BLACK = "\u001b[30;1m"
 
-  val default: Formatter = Formatter.create(ThrowableFormat.Default, colorful = false)
+  val default: Formatter = Formatter.create(ThrowableFormat.Default, PositionFormat.Full, colorful = false)
 
-  val colorful: Formatter = Formatter.create(ThrowableFormat.Default, colorful = true)
+  val colorful: Formatter = Formatter.create(ThrowableFormat.Default, PositionFormat.Full, colorful = true)
 
   /**
     * Creates new formatter with provided options
     *
     * @param throwableFormat @see [[formatThrowable]]
+    * @param positionFormat @see [[formatPosition]]
     * @param colorful use different color for thread name, level, position and throwable
     */
-  def create(throwableFormat: ThrowableFormat, colorful: Boolean): Formatter = {
+  def create(throwableFormat: ThrowableFormat, positionFormat: PositionFormat, colorful: Boolean): Formatter = {
```

Comment:
Ooops, you're breaking already released API here. Either create overloaded method `create` that takes three arguments, or put it in the end of arguments list and give it a default value

Reply:
I've added overloaded methods to `Formatter` and `circe.Formatter`.